validator.py

```diff
@@ -1,23 +1,58 @@
 import sys
+from jsonschema.exceptions import RefResolutionError
+from colors import color
 from os import path, getcwd
-from openapi_spec_validator import validate_spec_url
+from openapi_spec_validator.handlers import UrlHandler
+from openapi_spec_validator import openapi_v3_spec_validator
 
 
 def validate(file_name):
-    validate_spec_url('file://' + path.join(getcwd(), file_name))
+    counter = 0
+    try:
+
+        handler = UrlHandler('file')
+        spec = handler('file://' + path.join(getcwd(), file_name))
+        errors_iterator = openapi_v3_spec_validator.iter_errors(spec)
```

Comment:
Was trying this out and it fails if we have references in other files because the spec URL is not being sent (like in https://github.com/p1c2u/openapi-spec-validator/blob/7d3bc1d7ff13b6416b3311999aa42740efc5bfb5/openapi_spec_validator/shortcuts.py#L16)

Reply:
Did it work with the original code? Otherwise I would handle that in a separate issue

Reply:
Yeap, it did, mostly because I was using the provided wrapper (that encapsulates the `UrlHandler`